centos/interface.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+function containerdisks::needs_update() {
+  local name=centos
+  local tag="$1"
+  local base_url="https://cloud.centos.org/centos/${tag}/x86_64/images/"
+
+  local os_image="$(wget -O - "${base_url}" | grep "GenericCloud.*qcow2" | sed -e 's/.*>\(.*qcow2\)<.*/\1/' | sort -r | head -n 1)"
+  local repo="quay.io/containerdisks/centos"
+
+
+  local latest_shasum="$(wget -O - "${base_url}CHECKSUM" | grep ${os_image} | tail -n 1 | grep -o '[a-Z0-9]*$')"
```

Comment:
I see that something is not right with this check as the tests report an empty shasum:

```
Latest shasum for centos:8: 
```

Reply:
For me this works. Do you have a hint how I can reproduce the issue?